src/server/routes/admin/nonces.ts

```diff
@@ -0,0 +1,169 @@
+import { Static, Type } from "@sinclair/typebox";
+import { FastifyInstance } from "fastify";
+import { StatusCodes } from "http-status-codes";
+import { eth_getTransactionCount, getRpcClient } from "thirdweb";
+import {
+  lastUsedNonceKey,
+  recycledNoncesKey,
+  sentNoncesKey,
+  splitLastUsedNonceKey,
+} from "../../../db/wallets/walletNonce";
+import { getChain } from "../../../utils/chain";
+import { normalizeAddress } from "../../../utils/primitiveTypes";
+import { redis } from "../../../utils/redis/redis";
+import { thirdwebClient } from "../../../utils/sdk";
+import { standardResponseSchema } from "../../schemas/sharedApiSchemas";
+import { walletWithAddressParamSchema } from "../../schemas/wallet";
+
+export const responseBodySchema = Type.Object({
+  result: Type.Array(
+    Type.Object({
+      walletAddress: Type.String({
+        description: "Backend Wallet Address",
+        examples: ["0xcedf3b4d8f7f1f7e0f7f0f7f0f7f0f7f0f7f0f7f"],
+      }),
+      onchainNonce: Type.String({
+        description: "Last mined nonce",
+        examples: ["0"],
+      }),
+      lastUsedNonce: Type.String({
+        description: "Last incremented nonce sent to the blockchain",
+        examples: ["0"],
+      }),
+      sentNonces: Type.Array(
+        Type.String({
+          examples: ["0"],
+        }),
+        {
+          description:
+            "Nonces that were successfully sent to the blockchain but not mined yet.",
+        },
+      ),
+      recycledNonces: Type.Array(
+        Type.String({
+          examples: ["0"],
+        }),
+        {
+          description:
+            "Nonces that were acquired but failed to be sent to the blockchain, waiting to be recycled or cancelled.",
+        },
+      ),
+      chainId: Type.Number({
+        description: "Chain ID",
+        examples: [80001],
```

Comment:
Use 80002 as an example, 80001 is Mumbai -> deprecated

Reply:
Also nit: put this filed next to wallet address because the `chainId + address` tuple defines a wallet.